tools/refguide_check.py

```diff
@@ -466,7 +466,7 @@ def resolve(name, is_label=False):
 
     token = '<RST-VALIDATE-SYNTAX-CHECK>'
 
-    docutils.core.publish_doctree(
+    docutils._core.publish_doctree(
```

Comment:
Incorrect change?

Reply:
Ah, yes - I left `docutils.core` as is but this one went through.

Reply:
Fixed.